package/CHANGELOG

```diff
@@ -133,6 +133,7 @@ Enhancements
     'protein' selection (#2751 PR #2755)
   * Added an RDKit converter that works for any input with all hydrogens
     explicit in the topology (Issue #2468, PR #2775)
+  * Added sort method to the atomgroup (Issue #2976) 
```

Comment:
Note these entries should be newest first.

Also please do add yourself to the author list for 2.0 (above) and AUTHORS.

Reply:
Added my name to author list for 2.0 and AUTHORS.

https://github.com/MDAnalysis/mdanalysis/pull/3188/commits/67bef3a3e1e2d76872dfae0b5b678d4802397877
https://github.com/MDAnalysis/mdanalysis/pull/3188/commits/aeca48da8de3be1a8570168792a9c2e937ee2c7f